InjectionBundle/SwiftInjection.swift

```diff
@@ -563,7 +567,8 @@ extension NSObject {
         let object = "@".utf16.first!
         while cls != nil && cls != NSObject.self && cls != NSURL.self {
             let className = NSStringFromClass(cls!)
-            if className.hasPrefix("_") || className.hasPrefix("NS") {
+            if className.hasPrefix("_") || className.hasPrefix("NS") ||
+                className.hasPrefix("WK") {
```

Comment:
Is `WK` prefixed classes known to cause issues?

Reply:
Yes, people using WKWebView seems to crash the sweep.

Reply:
Added a "final" change which allows you to inject, injection itself. Thanks for taking a look. Will tweet about "HotReloading" and wait for feedback then merge this.